### ISSUE TITLE
Installation fails because of mixpanel dependency!

### ISSUE THREAD

**Issue by ser1us** [docker/toolbox#133]
Tried to install dockertools with brew cask (I know not the default way). But installation fails either way because of this error message:
```
Failed to connect to api.mixpanel.com port 443: Connection refuse
```

```
==> installer: The upgrade failed (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.)
==>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
==>                                  Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed to connect to api.mixpanel.com port 443: Connection refuse
Error: Command failed to execute!

==> Failed command:
["/usr/bin/sudo", "-E", "--", "/usr/sbin/installer", "-pkg", "#<Pathname:/opt/homebrew-cask/Caskroom/dockertoolbox/1.8.1b/DockerToolbox-1.8.1b.pkg>", "-target", "/"]

==> Output of failed command:
installer: Package name is Docker Toolbox
installer: Upgrading at base path /
installer: The upgrade failed (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.)


==> Exit status of failed command:
#<Process::Status: pid 4641 exit 1>
```

**Comment by jeffdm** [comment]
This will be fixed as of 1.8.1c, where installing via command line will now longer contact mixpanel

**Comment by ser1us** [comment]
Awesome, thank you! :)

**Comment by eproxus** [comment]
Another use case is installing via GUI while having mixpanel connections blocked (for example via Little Snitch).

**Comment by jeffdm** [comment]
This should be fixed in 1.8.1c. Thanks again for the issue!

@eproxus thanks for chiming in!

**Comment by eproxus** [comment]
Thanks for the fix!
